components/Banner.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+const Banner = () => {
+  return (
+    <div>
+      <div className="bg-white">
+        <div
+          className="p-4"
+          style={{
+            background:
+              "repeating-linear-gradient(-55deg, #222, rgb(34, 34, 34) 10px, rgb(234, 179, 8) 10px, rgb(234, 179, 8) 20px)",
+          }}
```

Comment:
In TailwindCSS, this is handled in the `tailwind.config.js` file by extending the `theme`. It would look something like this (I have also included how to handle the fluid font clamp):

```js
// tailwind.config.js
module.exports = {
  content: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
  theme: {
    extend: {
      fontFamily: {
        montserrat: ['Montserrat', 'sans-serif'],
      },
      fontSize: {
        dynamic: 'clamp(1rem, 1vw, 2rem)',
      },
      backgroundImage: {
        'banner-gradient':
          'repeating-linear-gradient(-55deg, #222, rgb(34, 34, 34) 10px, rgb(234, 179, 8) 10px, rgb(234, 179, 8) 20px)',
      },
    },
  },
  plugins: [],
};
```

This will result in two new classes `text-dynamic` and `bg-banner-gradient` which can be used in the component instead of inline styles.

```jsx
{/* Banner.jsx */}
<div className="bg-white">
  <div className="p-4 bg-banner-gradient" />
  <div className="p-4 text-center text-gray-900 bg-yellow-500 font-bold text-dynamic">UNDER CONSTRUCTION</div>
</div>
```

I do not think this is important for this component since we shall delete it once the site is ready. I only shared it here in case the knowledge is helpful.

The PR looks good to me as-is so I will go ahead and approve it 👍.

Reply:
@antosan  yep thanks for sharing knowledge, it helped me alot.